### ISSUE TITLE
Fix Lists in the Turtle Parser

### ISSUE THREAD

**Issue by kaefer3000** [nxparser/nxparser#1]
``` ttl
@prefix ex: <http://ex.org/> .
</213> ex:hasMembers ( ex:SnoopDogg ex:NateDogg ex:WarrenG ) .
```

results in `Node[]`s with `null`s, cf. `TurtleParserTest2.java` in e7401dc8471bded82e44adbacac5125400dbf5d2.

**Comment by aharth** [comment]
Bug also comes up in http://www.w3.org/ns/prov.ttl

**Comment by kaefer3000** [comment]
[OWLAPI's Turtle parser](https://github.com/owlcs/owlapi/blob/24a5bf0e69f96d49e0720eeed198abba6156d630/parsers/src/main/java/org/semanticweb/owlapi/rdf/turtle/parser/TurtleParser.jj) can handle those lists and is also grammar-based, so we might want to switch to that one.

**Comment by kaefer3000** [comment]
Fixed in commits d5824709d4ac9ed9269faa31e87b76d2e1cd5d59 and 6218a7d3c2eff53771de21bf635813dab2da455d
